Mathlib/AlgebraicGeometry/Morphisms/FinitePresentation.lean

```diff
@@ -71,4 +74,67 @@ instance {X Y Z : Scheme.{u}} (f : X ⟶ Z) (g : Y ⟶ Z) [LocallyOfFinitePresen
     LocallyOfFinitePresentation (Limits.pullback.snd f g) :=
   MorphismProperty.pullback_snd _ _ inferInstance
 
+/-- **Chevalley's Theorem**: The image of a locally constructible set under a
+morphism of finite presentation is locally constructible. -/
+@[stacks 054K]
+nonrec lemma Scheme.Hom.isLocallyConstructible_image (f : X.Hom Y)
```

Comment:
Not sure what you need the `nonrec` for? `isLocallyConstructible_image` doesn't appear within the proof
```suggestion
lemma Scheme.Hom.isLocallyConstructible_image (f : X.Hom Y)
```

Reply:
`wlog` gets confused and introduce spurious variables without it.

Reply:
Oh! Can you minimise and report on Zulip?

Reply:
Probably you should at least add a comment about this in the PR before merging?